solidity/contracts/Coordinator.sol

```diff
@@ -167,18 +158,61 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
     emit NewServiceAgreement(serviceAgreementID, _requestDigest);
   }
 
+  function fulfillOracleRequest(
+    bytes32 _requestId,
+    bytes32 _data
+  )
+    external
+    isValidResponse(_requestId)
+    returns (bool)
+  {
+    callbacks[_requestId].responses[msg.sender] = uint256(_data);
+
+    Callback memory callback = callbacks[_requestId];
+    callbacks[_requestId].responseCount += 1;
+    uint256 responseCount = callbacks[_requestId].responseCount;
+    bytes32 sAId = callbacks[_requestId].sAId;
+    address[] memory oracles = serviceAgreements[sAId].oracles;
+    if (oracles.length > responseCount) {
+      return true;
+    }
+
+    uint256 result;
+    uint256 oraclePayment = callback.amount.div(oracles.length);
```

Comment:
Something to think about, if the payment amount isn't cleanly divisible by the number of nodes then the remainder LINK will be locked in the contract without a withdrawable function.

Reply:
Not worth the gas costs to make a change, as the highest I've found that would be locked up is 11 wei. That's with 0.2 ether as payment with 27 oracles. Although, worth mentioning.

```solidity
contract PaymentRemainderTest {
    function oraclePayment(uint amount, uint oracleCount) public pure returns (uint256) {
        uint256 payment = amount / oracleCount;
        uint256 lostLink = amount - (payment * oracleCount);
        return lostLink;
    }
}
```